ext/bert/c/decode.c

```diff
@@ -14,9 +15,12 @@
 #define ERL_BIN           109
 #define ERL_SMALL_BIGNUM  110
 #define ERL_LARGE_BIGNUM  111
+#define ERL_ENC_STRING    112
+#define ERL_UNICODE_STRING 113
```

Comment:
Assuming the [online docs](http://erlang.org/doc/apps/erts/erl_ext_dist.html) are up to date, numbers from 120 onwards are free, but they could probably add new types at any point.

We could reduce the divergence by using 107 (`STRING_EXT`) for utf8 strings whose length fits within 2 bytes and go with `ENC_STRING` if they're bigger, but we'd still re-use or hope we never find new types useful for `ENC_STRING`.

The types we're making unavailable aren't something that I think make sense to transfer between ruby and erlang, so I don't think anybody will miss them. But even so making an note of the rationale in the code would be useful.